app/shared/availability-view/availability-view.less

```diff
@@ -86,14 +86,51 @@
       z-index: 3;
     }
     .react-date-field__calendar-icon {
-      display: none;
+      z-index: 10;
+      border: none;
+      &:hover {
+        background-color: transparent;
+        .react-date-field__calendar-icon-inner {
+          background-color: transparent;
+        }
+      }
+      &:active {
+        border: none;
+      }
+      &::before, &::after {
+        visibility: hidden;
+      }
+      .react-date-field__calendar-icon-inner {
+        line-height: @av-header-height;
+        height: @av-header-height;
+        margin: 0;
+        padding: 0;
+        width: 150px;
+        text-align: right;
+        cursor: pointer;
+        margin-left: -120px;
+        z-index: 10;
+        margin-left: -140px;
+        position: static;
+        background-color: transparent;
+
+        &::after {
+          font-family: 'Glyphicons Halflings';
+          content: "\e252";
+          visibility: visible;
+          color: white;
+          font-size: 18px;
+          position: absolute;
+          right: 0;
```

Comment:
- [ ] You could add about 20px of right, so the icon is not that far way from the input.